src/helpers/tsReader.ts

```diff
@@ -409,3 +408,25 @@ export const registerUserTs = (basePath: string): (() => void) | null => {
     throw new Error(`Error parsing your tsconfig.json file: ${(err as Error).message}`);
   }
 };
+
+export function parseTSConfig(tsConfigString: string, basePath: string) {
```

Comment:
Small cleanup, we can change the params here to simply accept `foundPath` (maybe rename to `tsconfigFilePath`) and then let this function call `const tsConfigString = fs.readFileSync(foundPath, "utf8")` and `path.dirname(extendedPath)`. This way you dont have to repeat it at line 421 and 422

Reply:
Great catch!